apps/docs/docs/learn/05-theming/02-using-variables.mdx

```diff
@@ -40,6 +40,19 @@ export const colors = stylex.defineVars({
   fontFamily: 'system-ui, sans-serif',
   fontSize: '16px',
 });
+
+const grid = 4;
+const px = (value: string | number) => `${value}px`;
```

Comment:
Congratulations on finding this hidden feature of StyleX! According to our documentation, you're not allowed to use functions to define styles.

We're not ready to document this feature. It's still experimental.

Reply:
Based on your guidance, I will revise this part to align with the established practices and avoid using experimental features not yet ready for documentation.